src/core.jl

```diff
@@ -112,16 +120,17 @@ one(n::Node) = one(unbox(n))
 # Leafs do nothing, Branches compute their own sensitivities and update others.
 @inline propagate(y::Leaf, rvs_tape::Tape) = nothing
 function propagate(y::Branch, rvs_tape::Tape)
-    tape = rvs_tape.tape
-    ȳ, f = tape[y.pos], y.f
-    xs, xids = map(unbox, y.args), map(pos, y.args)
-    p = preprocess(f, y.val, ȳ, xs...)
+    tape_ = tape(rvs_tape)
```

Comment:
Couldn't this itself be `tape = Nabla.tape(rvs_tape)`?

Reply:
Yeah it could, I actually realized that after I had pushed.